core_types/src/lib.rs

```diff
@@ -32,6 +32,9 @@ extern crate maplit;
 
 extern crate uuid;
 
+extern crate chrono;
+extern crate guid_create;
+
 pub mod cas;
 pub mod chain_header;
 pub mod crud_status;
```

Comment:
why the permissions and line ending changes in the files below?

Reply:
cargo fmt so line endings are affected differently on different systems. Windows vs linux

Reply:
kk